### PR TITLE
[ActionCable] Test available actions on Channel

### PR DESCRIPTION
**Pull request by akshay-vishnoi**

### CODE REVIEW

File: actioncable/test/channel/base_test.rb [hunk 1]
```diff
@@ -145,4 +145,8 @@ def rm_rf
     assert_equal expected, @connection.last_transmission
   end
 
+  test "actions available on Channel" do
+    available_actions = ["room", "last_action", "subscribed", "unsubscribed", "toggle_subscribed", "leave", "speak", "subscribed?", "get_latest", "chatters", "topic"].to_set
```
Comment (kaspth) [inline actioncable/test/channel/base_test.rb]: Could do `%w(room last_action ...)` to save some characters.

Comment (kaspth) [inline actioncable/test/channel/base_test.rb]: (But don't bother now that it's merged :smile:)

Comment (akshay-vishnoi) [inline actioncable/test/channel/base_test.rb]: I will change it in next PR taking stab at https://github.com/rails/rails/pull/22664/files#r48028620

**Pull request merged**